lib/ansible/modules/system/mksysb.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: Kairo Araujo (@kairoaraujo)
+module: mksysb
+short_description: Generates AIX mksysb rootvg backups.
+description:
+  - This module manages a basic AIX mksysb (image) of rootvg.
+version_added: "2.5"
+options:
+  backup_crypt_files:
+    description:
+      - Backup encrypted files.
+    choices: ["yes", "no"]
+    default: "yes"
+  backup_dmapi_fs:
+    description:
+      - Back up DMAPI filesystem files.
+    choices: ["yes", "no"]
+    default: "yes"
+  create_map_files:
+    description:
+      - Creates a new MAP files.
+    choices: ["yes", "no"]
+    default: "no"
+  exclude_files:
+    description:
+      - Excludes files using /etc/rootvg.exclude.
+    choices: ["yes", "no"]
+    default: "no"
+  exclude_wpar_files:
+    description:
+      - Excludes WPAR files.
+    choices: ["yes", "no"]
+    default: "no"
+  extended_attrs:
+    description:
+      - Backup extended attributes.
+    choices: ["yes", "no"]
+    default: "yes"
+  name:
+    description:
+      - Backup name
+    required: true
+  new_image_data:
+    description:
+      - Creates a new file data.
+    choices: ["yes", "no"]
+    default: "yes"
+  software_packing:
+    description:
+      - Exclude files from packing option listed in
+        /etc/exclude_packing.rootvg.
+    choices: ["yes", "no"]
+    default: "no"
+  storage_path:
+    description:
+      - Storage path where the mksysb will stored.
+    required: true
+  use_snapshot:
+    description:
+      - Creates backup using snapshots.
+    choices: ["yes", "no"]
+    default: "no"
+'''
+
+EXAMPLES = '''
+- name: Running a backup image mksysb
+  mksysb:
+    name: myserver
+    storage_path: /repository/images
+    exclude_files: yes
+    exclude_wpar_files: yes
+'''
+
+RETURN = '''
+changed:
+  description: Return changed for mksysb actions as true or false.
+  returned: always
+  type: boolean
+  version_added: 2.5
+msg:
+  description: Return message regarding the action.
+  returned: always
+  type: string
+  version_added: 2.5
+'''
+
+
+from ansible.module_utils.basic import AnsibleModule
+import os
+
+# Command options.
+map_file_opt = {
```

Comment:
I would put all these variables inside the `main()` function as there is no reason to have them global because you don't share them across multiple functions.